deploy/create-bridge-token-proposal.ts

```diff
@@ -0,0 +1,226 @@
+      tokenAmount,
+    ])
+  );
+
+  console.log(
+    "  Step 3b: Encode setWindow(amount, bridgedToken, merkleRoot, ipfsHash) to MerkleDistributor"
+  );
+
+  const merkleDistributorInterface = new Interface([
+    {
+      type: "function",
+      name: "setWindow",
+      inputs: [
+        { name: "rewardsToDeposit", type: "uint256", internalType: "uint256" },
+        { name: "rewardToken", type: "address", internalType: "address" },
+        { name: "merkleRook", type: "bytes32", internalType: "bytes32" },
+        { name: "ipfsHash", type: "string", internalType: "string" },
+      ],
+      outputs: [],
+      stateMutability: "nonpayable",
+    },
+  ]);
+  multicallTargets.push(constants.addresses.MERKLE_DISTRIBUTOR);
+  multicallDatas.push(
+    merkleDistributorInterface.encodeFunctionData("setWindow", [
+      tokenAmount,
+      bridgedTokenAddress,
+      merkleRoot,
+      ipfsHash,
+    ])
+  );
+
+  /**
+   * LAYER 1
+   */
+
+  console.log("  Step 3c: Encode aggregate3(Call[] calls) to OwnedMulticall");
+  const multicallArtifact = await hre.artifacts.readArtifact("OwnedMulticall3");
+  const multicallInterface = new Interface(multicallArtifact.abi);
+  const multicallData = multicallInterface.encodeFunctionData("aggregate3", [
+    multicallTargets.map((target, i) => ({
+      target,
+      callData: multicallDatas[i],
+      allowFailure: false,
+    })),
+  ]);
+
+  console.log(
+    "  Step 3d: Estimate gas cost for multicall transaction (overestimate by 3x)"
+  );
+  const zkSyncAddress = await wallet.provider.getMainContractAddress();
+  const zkSyncContract = new Contract(
+    zkSyncAddress,
+    utils.ZKSYNC_MAIN_ABI,
+    l1Wallet
+  );
+  const gasPrice = (await l1Wallet.provider.getFeeData()).gasPrice * 3n;
+  const gasLimit = await wallet.provider.estimateL1ToL2Execute({
+    contractAddress: constants.addresses.OWNED_MULTICALL,
+    calldata: multicallData,
+    caller: utils.applyL1ToL2Alias(constants.addresses.L1_TIMELOCK),
+  });
+  let baseCost = await zkSyncContract.l2TransactionBaseCost(
+    gasPrice,
+    gasLimit,
+    utils.REQUIRED_L1_TO_L2_GAS_PER_PUBDATA_LIMIT
+  );
+
+  console.log("  Step 3e: Encode Cross chain transaction");
+  const l2TransactionData = zkSyncContract.interface.encodeFunctionData(
+    "requestL2Transaction",
+    [
+      constants.addresses.OWNED_MULTICALL,
+      0,
+      multicallData,
+      gasLimit,
+      utils.REQUIRED_L1_TO_L2_GAS_PER_PUBDATA_LIMIT,
+      [],
+      constants.addresses.OWNED_MULTICALL,
+    ]
+  );
+  targets.push(zkSyncAddress);
+  values.push(baseCost);
+  calldatas.push(l2TransactionData);
+
+  console.log("Step 4: Create proposal");
+  const proposalDescription = `Transfer ${formatEther(
+    tokenAmount
+  )} INCR to zkSync Era and deposit in MerkleDistributor deployed at ${
+    constants.addresses.MERKLE_DISTRIBUTOR
+  } for Genesis Orb holder reward distribution (Merkle root: ${merkleRoot}, IPFS hash: ${ipfsHash})`;
```

Comment:
We may want to reduce this proposal description. Not sure what max size is but better to keep it short and explain more in the off-chain proposal.

Maybe: "Transfer {tokenAmount} INCR to MerkleDistributor on Era"